scripts/compose

```diff
@@ -35,9 +35,6 @@ if [ ${OPENBALENA_HOST_NAME: -6} == ".local" ]; then
   INCLUDE_MDNS="-f ${BASE_DIR}/compose/mdns.yml"
 fi
 
-# show a warning to update your balena CLI tool...
-echo_bold_stderr "IMPORTANT: Please update your Balena CLI installation to version v12.38.5"
```

Comment:
`echo_bold_stderr` is defined a few lines above but no longer used after this line is removed -- can you please remove the function too?

Reply:
This warning was added less than half a year ago, I'm sure there'd be users who haven't updated open-balena since then. The warning is a bit misleading in 2 counts though: telling the user to update the CLI suggests that an incompatible CLI version was detected (no such detection takes place), and specifically requesting an update to version v12.38.5 suggests that no other version is acceptable. Better wording might be:
```
echo_bold_stderr "IMPORTANT: This version of open-balena requires balena CLI version v12.38.5 or above"
```

Reply:
@pdcastro requirements are listed on the [project's README](https://github.com/balena-io/open-balena#compatibility) and IMO we shouldn't have added the warning in the first place -- this is not the place to warn users for potential tool incompatibilities.